docs/design/features/hybrid-globalization.md

```diff
@@ -198,3 +198,115 @@ Web API does not expose locale-sensitive endsWith/startsWith function. As a work
+
+
+### OSX
+
+For OSX platforms we are using native apis instead of ICU data.
+
+**String comparison**
+
+Affected public APIs:
+- CompareInfo.Compare,
+- String.Compare,
+- String.Equals.
+
+The number of `CompareOptions` and `NSStringCompareOptions` combinations are limited. Originally supported combinations can be found [here for CompareOptions](https://learn.microsoft.com/dotnet/api/system.globalization.compareoptions) and [here for NSStringCompareOptions](https://developer.apple.com/documentation/foundation/nsstringcompareoptions).
+
+- `IgnoreSymbols` is not supported because there is no equivalent in native api. Throws `PlatformNotSupportedException`.
+
+- `IgnoreKanaType` is not supported because there is no equivalent in native api. Throws `PlatformNotSupportedException`.
+
+- `None`:
+
+`CompareOptions.None` is mapped to `NSStringCompareOptions.NSLiteralSearch`
+
+There are some behaviour changes. Below are examples of such cases.
+
+| **character 1** | **character 2** | **CompareOptions** | **hybrid globalization** | **icu** |                       **comments**                      |
+|:---------------:|:---------------:|--------------------|:------------------------:|:-------:|:-------------------------------------------------------:|
+|   `\u3042` あ   |   `\u30A1` ァ   |   None  |             1            |    -1   |     hiragana and katakana characters are ordered differently compared to ICU    |
+|   `\u304D\u3083` きゃ  |   `\u30AD\u30E3` キャ |     None     |             1            |    -1   | hiragana and katakana characters are ordered differently compared to ICU  |
+|   `\u304D\u3083` きゃ  |   `\u30AD\u3083` キゃ  |     None     |             1           |    -1   |  hiragana and katakana characters are ordered differently compared to ICU  |
+|   `\u3070\u3073\uFF8C\uFF9E\uFF8D\uFF9E\u307C` ばびﾌﾞﾍﾞぼ  |   `\u30D0\u30D3\u3076\u30D9\uFF8E\uFF9E` バビぶベﾎﾞ  |     None     |   1  |  -1  | hiragana and katakana characters are ordered differently compared to ICU   |
+|   `\u3060` だ  |   `\u30C0` ダ  |     None     |   1  |  -1  |   hiragana and katakana characters are ordered differently compared to ICU |
+|   `\u00C0` À  |   `A\u0300` À  |     None     |   1  |  0  |   This is not same character for native api |
+
+- `StringSort` shares the same logic as None.
+
+- `IgnoreCase`:
+
+`CompareOptions.IgnoreCase` is mapped to `NSStringCompareOptions.NSCaseInsensitiveSearch`
+
+There are some behaviour changes. Below are examples of such cases.
+
+| **character 1** | **character 2** | **CompareOptions** | **hybrid globalization** | **icu** |                       **comments**                      |
+|:---------------:|:---------------:|--------------------|:------------------------:|:-------:|:-------------------------------------------------------:|
+|   `\u20A9` ₩  |   `\uFFE6` ￦  |     IgnoreCase     |   0  |  -1  |    |
```

Comment:
Huh, I would expect that this would be only considered equal if we passed in `IgnoreWidth` because the first is the Won Sign and the second is the Fullwidth Won Sign right? 
Same for the one right below

Reply:
After mapping `IgnoreCase` to `NSStringCompareOptions.NSCaseInsensitiveSearch | NSStringCompareOptions.NSLiteralSearch` this test case behaves like in `ICU`. Removed from doc.

Reply:
It's still a bit strange that just `NSStringComparisonOptions.NSCaseInsensitiveSearch` led to equivalence between `\u20A9 ₩ ` and  `\uFFE6 ￦`.

Reply:
This is not there anymore. Please refresh the file.

Reply:
Right, I was just noting it was odd to me how NSString handled that comparison :)